bitcoin/src/crypto/key.rs

```diff
@@ -116,8 +116,7 @@ impl PublicKey {
                 hash160::Hash::hash(&self.inner.serialize()).to_byte_array(),
             ))
         } else {
-            // We can't create witness pubkey hashes for an uncompressed
-            // public keys
+            // We can't create witness pubkey hash for an uncompressed pubkey.
```

Comment:
In 76f9ab2df76d5a65ade52c339d51a2999b8446ab

Should be "a witness pubkey hash".

Reply:
Done, thanks.